ethstorage/p2p/protocol/syncclient.go

```diff
@@ -1142,20 +1150,52 @@ func (s *SyncClient) report(force bool) {
 		if !t.done {
 			taskRemain++
 		}
+	}
+
+	etaSecondsLeft := totalSecondsUsed * blobsToSync / synced
+
+	// Create a mega progress report
+	var (
+		progress    = fmt.Sprintf("%.2f%%", float64(synced)*100/float64(blobsToSync+synced))
+		tasksRemain = fmt.Sprintf("%d@%d", taskRemain, subTaskRemain)
+		blobsSynced = fmt.Sprintf("%v@%v", log.FormatLogfmtUint64(synced), syncedBytes.TerminalString())
```

Comment:
Might be more common to use `/` instead of `@` to stand for ratio or fraction?

Reply:
this is same as go-ethereum